File: condicoes/1035.py
#1035 TESTE DE SELEÇÃO 1
valores = input().split(" ")

valorA = int(valores[0])
valorB = int(valores[1])
valorC = int(valores[2])
valorD = int(valores[3])

if (valorB > valorC and 
    (valorD > valorA) and 
    ((valorC+valorD) > (valorA+valorB)) and 
    (valorC >= 0 and valorD >= 0) and 
    (valorA%2 == 0)):
    print("Valores aceitos")
else:
    print("Valores nao aceitos")

File: javascript/1035.js
/*Teste de Seleção 1*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let lines = input.split(" ");

let A = parseInt(lines.shift());
let B = parseInt(lines.shift());
let C = parseInt(lines.shift());
let D = parseInt(lines.shift());


if(B > C &&
   (D > A) &&
   (C + D) > (A + B) &&
    (C >= 0) && 
    (D >= 0) &&
     A % 2 === 0) {
          console.log("Valores aceitos");
} else {                        
  console.log("Valores nao aceitos");   
}